### PR TITLE
Fixing issue with getentryassembly being null in case of .net2.0

### PR DESCRIPTION
**Pull request by kadurga**
GetEntryAssembly is null in case of .net2.0, which is causing failures in test code which uses reflections.
Fixing code to not run when GetEntryAssembly is null.

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/DocumentClient.cs [hunk 1]
```diff
@@ -811,137 +811,144 @@ private async Task OpenPrivateAsync(CancellationToken cancellationToken)
             DefaultTrace.InitEventListener();
 
 #if !(NETSTANDARD15 || NETSTANDARD16)
-            // For tests we want to allow stronger consistency during construction or per call
-            string allowOverrideStrongerConsistencyConfig = System.Configuration.ConfigurationManager.AppSettings[DocumentClient.AllowOverrideStrongerConsistency];
-            if (!string.IsNullOrEmpty(allowOverrideStrongerConsistencyConfig))
+#if NETSTANDARD20
+            if (System.Reflection.Assembly.GetEntryAssembly() != null)
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/src/DocumentClient.cs]: Please comment on purpose for reference.
Also is there a tracking item for master?

Comment (kadurga) [reply Microsoft.Azure.Cosmos/src/DocumentClient.cs]: Have already sent PR for master. Will add comment

**Comment by kadurga** [comment]
Can someone approve?

**Comment by kirankumarkolli** [comment]
Duplicate of #e5cab4c3cad6acc51e09ec101f4bcbf4c937ad64

**Pull request closed without merging**